lib/lolcommits/platform.rb

```diff
@@ -0,0 +1,111 @@
+# -*- encoding : utf-8 -*-
+require 'mini_magick'
+
+module Lolcommits
+  class Platform
+    def self.platform
+      if platform_fakeplatform?   then ENV['LOLCOMMITS_FAKEPLATFORM']
+      elsif platform_fakecapture? then 'Fake'
+      elsif platform_mac?         then 'Mac'
+      elsif platform_linux?       then 'Linux'
+      elsif platform_windows?     then 'Windows'
+      elsif platform_cygwin?      then 'Cygwin'
+      else
+        fail 'Unknown / Unsupported Platform.'
+      end
+    end
+
```

Comment:
maybe have `RUBY_PLATFORM.to_s.downcase` in a method, and use that in the 4 methods below?